src/modules/requirements/StarterRequirement.ts

```diff
@@ -0,0 +1,19 @@
+import { AchievementOption, Region } from '../GameConstants';
+import Requirement from './Requirement';
+
+export default class StarterRequirement extends Requirement {
+    region: Region;
+    constructor(region: Region, starterIndex) {
+        super(starterIndex, AchievementOption.equal);
+        this.region = region;
+    }
+
+    public getProgress() {
+        return player.regionStarters[this.region]();
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    public hint(): string {
+        return 'Wrong starter.';
```

Comment:
Could we change this to something like `Requires ${starter} to be chosen as your ${region} starter Pokémon`.
I know it won't likely be shown anywhere, but may end up being used by the bot for displaying some requirements later.

Reply:
Marked resolved but doesn't look like there were changes to resolve it?

Reply:
I don't know what happened. Stuff goes wrong when you spend more than 4 months on a PR. But should be fixed now